scripts/gh-projects/examples/matchline/phase-2/c8.md

```
@@ -0,0 +1,27 @@
+Per PRD § Technical principles: "The user's Experience Units belong to the user. Export to JSON is a V1 feature, not V2."
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)